web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -198,11 +198,18 @@ export namespace MenuOverlay {
 
           if (this.triggerElement) {
             this.triggerElement.setAttribute("aria-expanded", "true");
+            if(this.triggerElement.hasAttribute("ariaexpanded")){
+              this.triggerElement.setAttribute("ariaexpanded", "true");
+            }
           }
         } else {
           this.dispatchMenuClose();
           if (this.triggerElement) {
             this.triggerElement.removeAttribute("aria-expanded");
+
+            if(this.triggerElement.hasAttribute("ariaexpanded")){
```

Comment:
What is the difference between aria-expanded and ariaexpanded ?

Reply:
the trigger element can have aria-expanded or ariaexpanded incase of custom webcomponent which takes this as a property, that't why handling the case for this